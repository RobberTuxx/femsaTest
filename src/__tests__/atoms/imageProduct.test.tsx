import React from 'react';
import renderer from 'react-test-renderer';
import ImageProduct from "../../components/atoms/imageProduct";
it('render image product', ()=> {
    const component = renderer.create(<ImageProduct url={"https://loremflickr.com/640/480/business"}/>).toJSON()
    expect(component).toMatchSnapshot()
})
