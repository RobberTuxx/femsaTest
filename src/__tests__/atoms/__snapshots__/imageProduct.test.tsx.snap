// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render image product 1`] = `
<Image
  source={
    {
      "uri": "https://loremflickr.com/640/480/business",
    }
  }
  style={
    {
      "borderRadius": 10,
      "height": 86,
      "width": 86,
    }
  }
/>
`;
